lib/phoenix/endpoint/cowboy_handler.ex

```diff
@@ -104,9 +122,5 @@ defmodule Phoenix.Endpoint.CowboyHandler do
     end
   end
 
-  defp info(scheme, endpoint, config) do
```

Comment:
Same thing here. endpoint.url != scheme/endpoint being started. For example, we can start the same endpoint under https and http, with your changes, it wll show the same url though, because `endpoint.url` would always be the same.